Jenkinsfile

```diff
@@ -0,0 +1,3 @@
+Map pipelineParams = [ "mavenAdditionalArgs" : "-U -Djava.net.preferIPv4Stack=true -Dsurefire.rerunFailingTestsCount=5 -Dmaven.test.failure.ignore=true" ]
```

Comment:
It probably works without any of these additional args, have you tested just in case?